pkg/kafka/auth/options.go

```diff
@@ -82,6 +84,11 @@ func addKerberosFlags(configPrefix string, flagSet *flag.FlagSet) {
 		configPrefix+kerberosPrefix+suffixKerberosKeyTab,
 		defaultKerberosKeyTab,
 		"Path to keytab file. i.e /etc/security/kafka.keytab")
+	flagSet.Bool(
+		configPrefix+kerberosPrefix+suffixKerberosDisablePAFXFAST,
+		defaultKerberosDisablePAFXFast,
+		"Disable FAST negotiation when not supported by KDC's like Active Directory")
```

Comment:
Q: why are the variables in the code have additional "PA FX" prefix? Is there semantic difference between PF-FX-FAST and just FAST? (it would be useful to include a link to the spec in the help text)

Reply:
So the additional PA FX refers to the cookie used. I considered naming the variable like that because they were done the same way in the sarama library. There is not a logical difference between both so we can consider naming it FAST. Whats your say?